lascli/parser/organizations.py

```diff
@@ -7,8 +7,12 @@ def get_organization(las_client: Client, organization_id):
     return las_client.get_organization(organization_id)
 
 
-def update_organization(las_client: Client, organization_id, **optional_args):
-    return las_client.update_organization(organization_id=organization_id, **optional_args)
+def update_organization(las_client: Client, organization_id, payment_method_id, **optional_args):
+    return las_client.update_organization(
+        organization_id=organization_id, 
+        payment_method_id=payment_method_id,
+        **optional_args,
+    )
```

Comment:
is it necessary to create a new variable for this, or can we just use optional args for payment_method_id as well?